CONTRIBUTING.md

```diff
@@ -146,13 +146,13 @@ Guidelines for repo maintainers.
 
 We use assignee to denote PR ownership. If you are the assignee, then you should have the PR on your todo list until you merge or close it.
 - **Assign yourself** to a PR if you have the time to take on the responsibilities of ownership (described below).
-- **Don't assign others** to a PR. Feel free to ask someone to take ownership, but respect others time restrictions.
+- **Don't assign others** to a PR. Feel free to ask someone to take ownership, but respect others' time restrictions.
 - **Avoid assignee=author**. In the case where the PR author is also a maintainer, we will strive to have another maintainer own and merge the PR to ensure these steps are followed fairly by all.
 
 The assignee of a PR is responsible for:
 - **being the primary contact** for the PR author. Be polite; you're the face of the community to this contributor.
 - **managing the PR's labels**. Add `Needs: Author Input` or `Needs: Review` as necessary.
-- **ensuring the PR doesn't get stuck**. Avoid leaving the author wondering about the state of the PR. If you don't have time right now, saying "I'm a little swamped now but will try to get to this in \_" is better than radio silence for a week.
+- **ensuring the PR doesn't get stuck**. Avoid leaving the author wondering about the state of the PR. If you don't have time right now, saying "I'm a little swamped now but will try to get to this in" is better than radio silence for a week.
```

Comment:
```suggestion
- **ensuring the PR doesn't get stuck**. Avoid leaving the author wondering about the state of the PR. If you don't have time right now, saying "I'm a little swamped now but will try to get to this in \_" is better than radio silence for a week.
```
I think the backslash here is just escaping the `_` so that if another `_` is added somewhere the text between doesn't become italics.